egs/librispeech/asr1/local/run_spm.sh

```diff
@@ -241,20 +272,24 @@ if [ ${stage} -le 5 ]; then
         # feature extraction
```

Comment:
This feature extraction part in the recognition was changed. Please check the diff of `run.sh` in https://github.com/espnet/espnet/blob/master/egs/librispeech/asr1/run.sh
Now basic feature dump was moved to the stage 1, and you don't need to use `feats` now.